### PR TITLE
Deybyr647 poem

### PR DESCRIPTION
**Pull request by deybyr647**
Here is a minimalist poem, written in JavaScript. Copy the code and run it in the Chrome Dev Console !

### CODE REVIEW

File: deybyr647-poem.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+let message = `This poem is by ${author.firstName} ${author.lastName}. Enjoy`
+
+let poem = ["Roses Are Red", "Violets Are Blue", "I love JavaScript", "I hope you do too!"]
+
+let poemOut = () => {
+  let i;
+  for(i=0; i <= poem.length; i++){
+    console.log(poem[i])
+    }
```
Comment (jkschulz) [inline deybyr647-poem.js]: Please put the brace at the same indent level as line 9

**Review by jkschulz — CHANGES_REQUESTED** [review]
Fantastic! Great description in the PR too. One minor suggestion

File: deybyr647-poem.js (hunk 1 above)
Comment (deybyr647) [reply deybyr647-poem.js]: Alright, I'll get to it now

**Comment by ssbalbuena** [comment]
wonderful!

**Review by jkschulz — APPROVED** [review]

**Pull request merged**